client/components/HistoryTotalOodlet.vue

```diff
@@ -30,13 +30,34 @@
 <script>
   import moment from 'moment';
   
-  export default{
+  export default {
+    data() {
+      return {
+        offices: []
+      }
+    },
+    
     props: ['totalOodlet'],
     
     computed: {
       orderedOn() {
         return moment(this.totalOodlet.orderedOn).locale('hr').format('LL');
       }
+    },
+    
+    methods: {
+      getOffices(){
```

Comment:
I love this dance :heart: 
1) I suggest something before we start
2) You think it's overkill so we discard it
3) Somewhere along the line I have to make some ugly woodoo code to go around no. 2
4) You see ugly code and suggest my original idea
:smiling_imp: :bowtie: :smiley: